mailpoet/tests/performance/tests/subscribers-filtering.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-shadow */
```

Comment:
Is this necessary?

Reply:
I don't think it is, I could remove it if you think the same. Maybe some leftover from the beginning.

Reply:
But I found out that it is important to have in every test because it is used to disable eslint error for having 2 same `subscribersFiltering` names in the test, one for the name of the test and the second in the bottom export function.

Reply:
The linter is throwing the error because the inner function, the group callback, has the same function name as the top level function:

```js
export function subscribersFiltering() { // <-----------
  const browser = chromium.launch({
    headless: headlessSet,
    timeout: timeoutSet,
  });
  const page = browser.newPage();

  group('Subscribers - Filter subscribers', function subscribersFiltering() { // <-----------
```

I think it should be possible to use an anonymous arrow function like this, avoiding the error:

```js
group('Subscribers - Filter subscribers', () => {
```

This would also be more idiomatic for the style of javascript we have in the codebase.

Reply:
> group('Subscribers - Filter subscribers', () => {

This one worked fine and I made changes, thanks!